CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ## [Next version]
 
-## [5.2.0] - 2019-05-29
+## [5.2.1] - 2019-05-30
```

Comment:
Why is the version different here?

Reply:
https://github.com/onfido/onfido-sdk-ui/pull/673#discussion_r289322875